src/reducers/breakpoints.js

```diff
@@ -20,7 +20,7 @@ import type { Record } from "../utils/makeRecord";
 
 export type BreakpointsState = {
   breakpoints: I.Map<string, Breakpoint>,
-  pendingBreakpoints: ?I.Map<string, any>,
+  pendingBreakpoints: any,
```

Comment:
this should be a `I.Map<string, PendingBreakpoint` -- for this we need to make a `PendingBreakpoint type`